### ISSUE TITLE
Stat size not matching file size

### ISSUE THREAD

**Issue by reintroducing** [th0r/webpack-bundle-analyzer#90]
#### Issue description
When I load up the server and see the breakdown, the total stat size is MUCH smaller than what my actual output file is, almost 1/4th the size (reporting 5.76mb, actual around 19.5mb [don't judge :)]). I'm trying to figure out if I'm misunderstanding how this tool works or if there is something wrong here. I'd like to see the real reflection of size because then it would give me a better idea of true size of each item in the bundle. Right now, though it may be proportional regardless, I can't tell exact size of the items in the bundle since overall the size is so much smaller.

#### Technical info
* Webpack Bundle Analyzer version: 2.8.2
* Webpack version: 3.0.0
* Node.js version: 6.9.2
* npm/yarn version: npm 3.10.9
* OS: OSX 10.12.4 Beta (16E154a)

#### Debug info
How do you use this module? As CLI utility or as plugin? 
Plugin

If CLI, what command was used? (e.g. `webpack-bundle-analyzer -O path/to/stats.json`)

If plugin, what options were provided? (e.g. `new BundleAnalyzerPlugin({ analyzerMode: 'disabled', generateStatsFile: true })`) 
No options provided, just the default.

What other Webpack plugins were used? 
```
new webpack.IgnorePlugin(/^\.\/locale$/, /moment$/),
new webpack.optimize.ModuleConcatenationPlugin(),
new webpack.DefinePlugin({
    'process.env': {
        NODE_ENV: JSON.stringify('production')
    }
}),
new webpack.optimize.UglifyJsPlugin({
    sourceMap: true,
    comments: false,
    compress: {
        screw_ie8: true,
        warnings: false
    },
    mangle: {
        screw_ie8: true
    }
})
```

It would be nice to also attach webpack stats file.
It can be generated using these options:
```js
new BundleAnalyzerPlugin({
  analyzerMode: 'disabled',
  generateStatsFile: true,
  // Excludes module sources from stats file so there won't be any sensitive data
  statsOptions: { source: false }
})`
```
`stats.json` will be created in Webpack bundle output directory.
[stats.json.zip](https://github.com/th0r/webpack-bundle-analyzer/files/1115976/stats.json.zip)

**Comment by valscion** [comment]
Thanks for the issue! This does seem weird.

I'm not at my computer right now so I can't yet verify what you're seeing, hopefully I'll get to this later :)

I'll have to ask a silly question, sorry: Are you sure you're looking at the Stat sizes and not Parsed sizes in the UI? Hover your mouse over the left edge of the reporter UI and make sure you're using the Stat sizes.

Although, that choice shouldn't change what the tooltip says a module size is :/

**Comment by th0r** [comment]
I have a suspicion that it's because of the `ModuleConcatenationPlugin`. Will investigate it a little later.

**Comment by valscion** [comment]
Oh of course, that will make things much more difficult. I didn't notice you were using that. Can you try without the moduleconcatenation plugin and verify the sizes are correct without it, @reintroducing?

**Comment by th0r** [comment]
@reintroducing Looking at your issue right now. Are you talking about `sh.min.js` that is 5.76mb? And it's real size is ~19.5mb? Maybe it just contains inline source maps?

**Comment by reintroducing** [comment]
Ugh, @th0r you're absolutely right, really sorry about the noise. I was testing with our dev build and then output our production build and was comparing apples to oranges. The dev build has `eval-source-map` set and the production build is `source-map`. My apologies for the noise, I should have caught this myself.

I did notice that when running the dev build I only get the stat size and not the parsed or gzipped. I believe I looked over another issue where this was happening and the reason was because webpack dev server was running and there is no way to get that info unless you run the cli version, is that correct?

Regardless, thank you for this great tool, it has really opened my eyes into what's going on in that bundle and i think its fantastic that its smart enough to determine what part is code and what part is source map in the bundle. Great job.
